src/Common/Entity/Program.php

```diff
@@ -328,4 +329,20 @@ public function setEnableEmailLogin($enable_email_login)
     {
         $this->enable_email_login = $enable_email_login;
     }
+
+    /**
+     * @return mixed
+     */
+    public function getTimezone()
+    {
+        return $this->timezone;
+    }
+
+    /**
+     * @param  mixed  $timezone
+     */
+    public function setTimezone($timezone): void
```

Comment:
This might be a problem, unless we're only supporting 7.4/8 moving forward -- i guess we lock in composer so it's probably OK?

Reply:
This is all good, disregard.